setup.py

```diff
@@ -32,6 +35,16 @@
 tf_cpu = 'tensorflow'
 tf_gpu = 'tensorflow-gpu'
 
+
+def latest_version(package_name):
+    url = "https://pypi.python.org/pypi/%s/json" % (package_name,)
```

Comment:
```suggestion
    url = f"https://pypi.python.org/pypi/{package_name}/json"
```
python 3.6 brings the lovely format strings :)